### PR TITLE
Prune Oban jobs older than a week 

### LINKED ISSUE #776: Suggestion to use Oban.Plugins.Pruner
Currently rows of oban_jobs table increases monotonically, so I suggest to use [Oban.Plugins.Pruner](https://hexdocs.pm/oban/Oban.Plugins.Pruner.html).

This plugin does

> Periodically delete completed, cancelled and discarded jobs based on age.

I can make a simple PR for this, but before that I would like to ask the development team what the duration of the max_age option should be.

### PR DESCRIPTION
**Pull request by joshk**
resolves #776

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -36,6 +36,7 @@ config :nerves_hub, Oban,
   log: false,
   queues: [delete_firmware: 1, firmware_delta_builder: 2, truncate: 1],
   plugins: [
+    {Oban.Plugins.Pruner, max_age: 3_600},
```
Comment (jjcarstens) [inline config/config.exs:39]: And hour seems aggressive? I can see a case where I want to see some of the jobs history for longer than an hour. Thoughts?

**Comment by joshk** [comment]
The default is 1 min.

What about 24 hours? 3 days?

You pick.

**Comment by jjcarstens** [comment]
O geez. lol. How about 7 days? We don't use oban a whole lot and that gives introspection time to start with

File: config/config.exs (hunk 1 above)
Comment (joshk) [inline config/config.exs:39]:
```suggestion
    {Oban.Plugins.Pruner, max_age: 604_800}, # 1 week 
```

**Comment by joshk** [comment]
@jjcarstens  review/approval please

**Review by jjcarstens — APPROVED** [review]

File: config/config.exs [hunk 2]
```diff
@@ -36,6 +36,7 @@ config :nerves_hub, Oban,
   log: false,
   queues: [delete_firmware: 1, firmware_delta_builder: 2, truncate: 1],
   plugins: [
+    {Oban.Plugins.Pruner, max_age: 604_800}, # 1 week
```
Comment (joshk) [inline config/config.exs:39]:
```suggestion
    # 1 week
    {Oban.Plugins.Pruner, max_age: 604_800},
```

**Pull request merged**